include/swift/Basic/Located.h

```diff
@@ -2,16 +2,15 @@
 //
 // This source file is part of the Swift.org open source project
 //
-// Copyright (c) 2014 - 2019 Apple Inc. and the Swift project authors
+// Copyright (c) 2019 Apple Inc. and the Swift project authors
 // Licensed under Apache License v2.0 with Runtime Library Exception
 //
 // See https://swift.org/LICENSE.txt for license information
 // See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
 //
 //===----------------------------------------------------------------------===//
 //
-// This file forward declares and imports various common LLVM datatypes that
-// swift wants to use unqualified.
+// Provides a currency data type Located<T> that should be used instead of std::pair<T, SourceLoc>.
```

Comment:
Sorry, when I posted the doc comment example, I didn't check the line width. Could you wrap this doc comment and the later ones to < 80 chars?

Reply:
Sure. Updated.